presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -279,6 +280,15 @@ public SystemSessionProperties(
                         "Experimental: enable iterative optimizer",
                         featuresConfig.isIterativeOptimizerEnabled(),
                         false),
+                new PropertyMetadata<>(
+                        ITERATIVE_OPTIMIZER_TIMEOUT,
+                        "Timeout for plan optimization in iterative optimizer (0 - timeout disabled)",
```

Comment:
Hmm... using 0 to mean "disabled" is weird and inconsistent. Unfortunately, I realized there's no way to set a session property to "missing" (unlike configuration options). In light of this, I think it'd be better to not give 0 any special meaning and set the default to a very large number that should never happen in practice (in the order of several minutes. Say, somewhere between 1 and 10 minutes).